source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2476,6 +2476,49 @@ export class ImageSet extends CardElementContainer {
     }
 }
 
+export class CaptionSource extends SerializableObject {
+    //#region Schema
+
+    static readonly mimeTypeProperty = new StringProperty(Versions.v1_1, "mimeType");
+    static readonly urlProperty = new StringProperty(Versions.v1_1, "url");
+
+    @property(CaptionSource.mimeTypeProperty)
+    mimeType?: string;
+
+    @property(CaptionSource.urlProperty)
+    url?: string;
+
+    //#endregion
+
+    protected getSchemaKey(): string {
+        return "CaptionSource";
+    }
+
+    constructor(url?: string, mimeType?: string) {
+        super();
+
+        this.url = url;
+        this.mimeType = mimeType;
+    }
+
+    isValid(): boolean {
+        return this.mimeType && this.url ? true : false;
+    }
+
+    render(): HTMLElement | undefined {
+        let result: HTMLTrackElement | undefined = undefined;
+
+        if (this.isValid()) {
+            result = document.createElement("track");
+            result.src = this.url!;
+            result.kind = "captions";
+            result.label = this.mimeType!;
```

Comment:
As the specs say:
![image](https://user-images.githubusercontent.com/21356912/161477390-0cec096a-e7e4-49ca-80c3-7c90336d4880.png)

So label should reflect the **language** of the chosen track instead of its **format**: `vtt` or `srt` (for example).

Reply:
Added a language property, and label gets set to that now